dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/LogTraceData.java

```diff
@@ -264,88 +318,174 @@ public String getObjectIdKey() {
         return NAMES[20];
     }
 
-    public String getDatetimeKey1_1() {
-        return NAMES1_1[0];
+    public String getDatetimeKey1_1(String eventType) {
```

Comment:
since you're just using the parameter to differentiate log vs. trace events, should probably use a boolean here (eg. boolean isMessageEvent)